src/helm/benchmark/presentation/summarize.py

```diff
@@ -1340,6 +1343,12 @@ def main():
         default=None,
         help="EXPERIMENTAL: Full class name of the Summarizer class to use. If unset, uses the default Summarizer.",
     )
+    parser.add_argument(
+        "--validity-check",
+        type=bool,
+        help="Whether to load the validity signals as metrics to display",
```

Comment:
Indicate that this feature is experimental:

`help="EXPERIMENTAL: Whether..."`